waspc/cli/Command/Deps.hs

```diff
@@ -0,0 +1,27 @@
+module Command.Deps
+  ( deps,
+  )
+where
+
+import Command (Command)
+import Control.Monad.IO.Class (liftIO)
+import Generator.ServerGenerator as Server (waspNpmDeps)
+import qualified Generator.WebAppGenerator as WebApp (waspNpmDeps)
+import NpmDependency (printDep)
+import qualified Util.Terminal as Term
+
+deps :: Command ()
+deps =
+  liftIO $
+    putStrLn $
+      unlines $
+        [ "",
+          title "WASP DEPENDENCIES",
```

Comment:
Instead of `WASP DEPENDENCIES` piece of text, maybe we could put something more descriptive.

Suggestion: `Below are listed the dependencies that Wasp uses in your project. You can import and use these directly in the code as if you specified them yourself, but you can't change their versions.`. Hopefully this should provide some clarity on the dependencies!